apis/shared-dimensions/lib/shapes/hierarchy.ts

```diff
@@ -111,6 +110,20 @@ export default function (graph: AnyPointer): Initializer<NodeShape> {
       node: nextInHierarchyShape,
       minCount: 1,
       maxCount: 1,
-    }],
+    }]
+
+    if (rdfTypeProperty) {
+      property.push({
+        path: rdf.type,
+        hasValue: [hydra.Resource, meta.Hierarchy, md.Hierarchy],
+        [dash.hidden.value]: true,
+      })
+    }
```

Comment:
The cusp of this change is to add constraint to `rdf:type` property when updating a hierarchy. Not required when creating